plugins/node/opentelemetry-instrumentation-mysql/package.json

```
@@ -47,6 +47,7 @@
   },
   "devDependencies": {
     "@opentelemetry/api": "^1.3.0",
+    "@opentelemetry/sdk-metrics": "^0.32.0",
```

There is a stable metrics SDK release now :slightly_smiling_face: 
```suggestion
    "@opentelemetry/sdk-metrics": "^1.8.0",
```

---

done